docs/src/modules/sandbox/FlattenRelativeImports.ts

```diff
@@ -5,6 +5,9 @@ export default function flattenRelativeImports(rawCode: string, modulePaths: str
     // Move the relative import to the current directory
     const newPath = `./${pathWithoutExtension.replace(/^.*[\\/]/g, '')}`;
     newCode = newCode.replace(pathWithoutExtension, newPath);
+
+    // Replace the relative import towards parent directory
+    newCode = newCode.replace('../', './');
```

Comment:
We need to replace nested imports toward the parent directory, e.g. a file importing from "../utils". Not sure if this is the best way to fix this, but I am open for suggestions.